types/office-js-preview/index.d.ts

```diff
@@ -476,6 +516,114 @@ declare namespace Office {
         */
         value: T;
     }
+    /**
+     * Message used in the visibility mode changed event.
+     */
+    interface VisibilityModeChangedMessage {
+        /**
+         * Visibility changed state.
+         */
+        visibilityMode: Office.VisibilityMode;
+    }
+    /**
+     * Function type to turn off the event.
+     */
+    type RemoveEventListener = () => Promise<void>;
+    /**
+     * Represents add-in level functionality for operating or configuring various aspects of the add-in.
+     */
+    interface Addin {
+        /**
+         * Set the startup behavior for the add-in for when the document is opened next time.
+         * @param behavior Specifies startup behavior of the add-in.
```

Comment:
TSDoc prefers a hyphen between the parameter name and the description.
https://github.com/Microsoft/tsdoc